src/components/Flexbox/index.ts

```diff
@@ -1,3 +1,5 @@
 import { Flexbox } from "./Flexbox";
+import { FlexboxProps } from "styled-system";
```

Comment:
in this case aren't we importing FlexboxProps twice since it's also imported in Flexbox.tsx?

Reply:
I don't think so... but I've updated the code to add the export of the `FlexboxProps` in the `Flexbox.tsx` where it should be defined anyway... the code is just proxy the import and exporting it again (under our package namespace)

Reply:
the `index.ts` is the one the is called when you `import { Flexbox } from "@components/Flexbox`... we add this file to avoid `import { Flexbox from @components/Flexbox/Flexbox` duplication.